docs/plainyearmonth.md

```diff
@@ -20,16 +20,18 @@ A `Temporal.PlainYearMonth` can be converted into a `Temporal.PlainDate` by comb
 - `isoYear` (number): A year.
 - `isoMonth` (number): A month, ranging between 1 and 12 inclusive.
 - `calendar` (optional `Temporal.Calendar` or plain object): A calendar to project the month into.
-- `referenceISODay` (optional number): A reference day, used for disambiguation when implementing other calendar systems.
-  You can omit this parameter unless using a non-ISO-8601 calendar.
+- `referenceISODay` (optional for ISO 8601 calendar; required for other calendars): A reference day, used for disambiguation when implementing calendar systems.
+  For the ISO 8601 calendar, this parameter will default to 1 if omitted.
+  For other calendars, it's the callers responsibility to set this parameter to the ISO-calendar day corresponding to the first day of the desired calendar year and month.
 
-**Returns:** a new `Temporal.PlainYearMonth` object.
+> NOTE: To avoid infinite recursion, `referenceISODay` is accepted as-is without validating that the day provided is actually the first day of the month in the desired calendar system.
```

Comment:
Suggest similar changes to my comment above.

Reply:
Done!